test/cql-pytest/cassandra_tests/validation/operations/select_test.py

```diff
@@ -1307,7 +1307,7 @@ def testPKQueryWithValueOver64K(cql, test_keyspace):
                            "SELECT * FROM %s WHERE a = ?", too_big)
 
 # Reproduces #10366
-@pytest.mark.xfail(reason="#10366 - server error instead of silent return")
+@pytest.mark.xfail(reason="#10366 - server error instead of silent return - limitation of scylla serialized objects size")
```

Comment:
Let's change the wording a bit. First, it's no longer a server error but an InvalidRequest (please check!). Second, we should say that we probably never want to fix this, because we consider it a Cassandra bug. Perhaps something like this:
reason="Cassandra silently ignores oversized clustering key, Scylla chose to generate an error as in the case of an oversized partition key - see discussion in #10366".